.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Contact Tenzir
+    url: https://tenzir.com/contact
+    about: Reach out to Tenzir
+  - name: Gitter Chat
+    url: https://gitter.im/tenzir/chat
+    about: Please ask and answer questions here.
```

```suggestion
    about: Come chat with us interactively!
```

Just an idea for more pizzazz.